maintainers/maintainer-list.nix

```diff
@@ -3692,6 +3692,12 @@
     githubId = 1222362;
     name = "Matías Lang";
   };
+  criyle = {
+    email = "i+nixos@goj.ac";
+    name = "Yang Gao";
+    githubId = 6821729;
+    github = "criyle";
+  };
```

Comment:
Adding yourself to maintainers needs to be a separate commit; `maintainers: add criyle`

Reply:
Thanks, moved to a separate commit